.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,83 @@
+name: Report an issue in wallet
```

Comment:
Should say "the wallet", although I think using "Firefly" might be better anyhow.